demoextendtemplates/views/PrestaShop/Admin/layout.html.twig

```diff
@@ -0,0 +1,72 @@
+{#
+  Example to modify layout (lets say you want to add your own custom flash message).
+  It is important to follow the path for layout.html.twig as it is in prestashop.
+#}
+
+{# first, we extend the original layoyt.html.twig #}
+{% extends '@PrestaShopCore/Admin/layout.html.twig' %}
+
+{# Then, we override the block we want to change#}
+{% block session_alert %}
+  {# import _self as layout is important, else the layout variable wouldn't be recognized #}
+  {% import _self as layout %}
+  {# render the original content from parent #}
+  {{ parent() }}
+  {# add our own custom flash #}
+  {% if app.session.flashbag.peek('demoextendtemplates-success')|length() > 0 %}
+    {# you can modify the css of "my-custom-class" however you want in your module css files  #}
+    <div class="some-kind-of-wrapper-class-if-needed">
+        {{ layout.alert('demoextendtemplates-success', app.session.flashbag.get('demoextendtemplates-success')) }}
+    </div>
+  {% endif %}
+
+  {#
+    We can even override the macro and change how the layout.alert works inside.
+    Be aware that this will change the behavior for all the falsh messages.
```

Comment:
```suggestion
    Be aware that this will change the behavior of all the flash messages, which is probably not recommended unless you are absolutely sure it is needed.
```

Reply:
instead added note above that the whole layout.html.twig will affect all pages.